aubase410/Module.md

```diff
@@ -1,27 +1,28 @@
-# FHIR R4 AUBase module
+
+# FHIR R4 Base module
 
 ## Sample Usage
 
 This section focuses on samples depicting how to use this package to implement FHIR related integrations
 
 ### Prerequisites
 
-1. Install Ballerina 2201.5.0 (Swan Lake Update 5)
+1. Install Ballerina 2201.6.0 or later
 
 ### 1. Parse JSON FHIR resource to FHIR resource model
-Sample below is using the AUBasePatient resource in `health.fhir.r4.aubase410` package.
+Sample below is using the Patient resource in `health.fhir.r4.aubase410` package.
 
 ```ballerina
-import ballerinax/health.fhir.r4;
```

Comment:
the r4 import should be there to work this sample?

Reply:
fixed